### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by matthewclegg**

### CODE REVIEW

**Review by riemanli — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -116,36 +110,58 @@ DIR=<some-path-that-will-contain-data-and-results>
...
+  --experimental_design=driver/single_publisher_design.py \
+  --output_file=$DIR/results \
+  --intermediates_dir=$DIR/intermediates \
+  --cores=0
 ```
 
-This will generate an analysis of the two models against the ten data sets,
-which will be recorded in the file `$DIR/results`. You can then see an analysis
-of this by using the colab, `analysis/two_pub_analysis.ipynb`.
+Setting `cores=0` enables multithreading.  Even so, the above design takes very long to run, so you
```
Comment (jiayu-google) [inline README.md:159]: may clarify that "enables multithreading using all available cores."

**Review by jiayu-google — APPROVED** [review]
Suuuper clear.  Thank you Matt!

File: README.md (hunk 1 above)
Comment (matthewclegg) [reply README.md:159]: Done.

**Review by RonPinelliMRC — APPROVED** [review]
MRC approving based on cursory review after confirming non-independent code review on an exception basis. Experimental code will require independent review prior to publication of or reliance on results generated.

**Pull request merged**